tests/performance-test/performance-test.sh

```diff
@@ -52,28 +124,59 @@ do
 done
 
 if [ "${TOOL}" = "tg" ]; then
-    MSGS=$((HOSTS * PLUGINS))
-    LENGTH_S=$((COUNT / INTERVAL))
-    CONFIG="\
-- metadata:
-    name: SAF Performance Test 1
-  spec:
-    value-lists: ${MSGS}
-    hosts: ${HOSTS}
-    plugins: ${PLUGINS}
-    interval: ${INTERVAL}
-    length: ${LENGTH_S}"
-
-    echo "${CONFIG}" > deploy/config/generated-test-configs.yml
-    cd deploy
-    export TG_CONFIGFILE=./config/generated-test-configs.yml
-    ./performance-test-tg.sh
-
+    echo "Collectd-tg not implemented. Try running with '-t tb' instead"
+    exit 1
 elif [ "${TOOL}" = "tb" ]; then
-    export COUNT HOSTS PLUGINS INTERVAL CONCURRENT
-    cd deploy
-    ./performance-test-tb.sh
-
+    :
 else
     usage
 fi
+
+get_sources
+create_grafana_datasources
+create_service_monitors
+post_dashboards
+
+STAGE="TARGET"
+while true; do
+    case $STAGE in
+        "TARGET")
+            TARGETS=`curl "http://$(oc get route prometheus -o jsonpath='{.spec.host}')/api/v1/targets"`
```

Comment:
Is this the default Prometheus path that is created by the `deploy.sh`? I would expect this to fail at some point in the future since it's not using an `https` endpoint. Not sure if the `https` endpoint exists right now, so it's entirely possible this is the currently valid approach.

Future you could use `curl -k https://...` I think.

Reply:
It currently is valid because the Prometheus in SAF does not use https